frontend/src/metabase/visualizations/components/TableSimple/TableSimple.styled.tsx

```diff
@@ -42,14 +42,14 @@ export const Table = styled.table`
   ${standardTableStyleReset}
 
   tr {
-    border-bottom: 1px solid ${alpha(color("border"), 0.3)};
+    border-bottom: 1px solid ${() => alpha(color("border"), 0.3)};
```

Comment:
this is another place where we convert the expression to a function. I'd like to know what was the reason for that.